src/components/CmsObserver.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="observer" />
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+
+@Component({ name: 'cms-observer' })
+export default class CmsObserver extends Vue {
```

Comment:
I don't know if `CmsObserver` is descriptive enough? I wouldn't know what it observed without reading the code